app/helpers/organisation_helper.rb

```diff
@@ -0,0 +1,9 @@
+module OrganisationHelper
+  def options_for_organisation_select(selected_id: nil)
+    [{ text: "None", value: nil }] + Organisation.not_closed.map do |organisation|
```

Comment:
Now we're doing this in one place, can we remove the default sorting scope and sort here (and in the index page), or do think it's better to sort at database level? I suppose I'm wondering if it'll slow things down in situations where we don't need to sort but it's probably not a big deal.

Reply:
We could, but we already order a few other models using a default scope, and I don't think we have a use case for not wanting this anywhere yet. I've removed the `order(:name)` call from a few other places in the latest push, and moved the changes into the commit where the default scope is added to make the benefits/use cases clearer